apis/cloudbuild/v1alpha1/workerpool_types.go

```diff
@@ -68,7 +68,10 @@ type CloudBuildWorkerPoolStatus struct {
 	// +optional
 	ObservedGeneration *int64 `json:"observedGeneration,omitempty"`
 
-	ResourceID *string `json:"resourceID,omitempty"`
+	/* The GCP URL to the cloudbuild workerpool resource.
+	The cloudbuild workerpool does not have selfLink. */
+	// +optional
+	ExternalRef *string `json:"externalRef,omitempty"`
```

Comment:
Oooh I like this metaphor, mirroring `external` in a ref.  If we go with this, Eventually we'll want to make sure that this format is always accepted as a ref.external format (we can always support additional formats as well, but it'll be much less confusing for users and for us if we have them match)

Reply:
Yes!